### PR TITLE
Prompt Manager - Uday

### PR DESCRIPTION
**Pull request by UdayKalra**
Implement initial prompt creation suite and testing.

### CODE REVIEW

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.servlets;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public final class PromptManager {
+
```
Comment (jelares) [inline backstory/src/main/java/com/google/sps/servlets/PromptManager.java:21]: If you get a chance, please add some documentation about the class itself (what it represents, and how the internal rep [keywords] is used to represent it), even if it's trivial, just to follow the style guide.

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   */
+  public String generatePrompt() {
+    String prompt = "";
+    if(keywords.size() == 0){
+        return "";
+    }
+
+
+    for(int incrementer = 0; incrementer < keywords.size() - 1; incrementer++){
+        prompt = prompt + keywords.get(incrementer) + " ";
```
Comment (jelares) [inline backstory/src/main/java/com/google/sps/servlets/PromptManager.java:46]: Consider changing the delimiter character (" ") to be a parameter of the function, then you could get easily labels like:
"label1 and label2 and label3 ... " if we decide that would be slightly better for the prompt.

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
```
Comment (jelares) [inline backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:1]: NIT: Copyright 2020 Google LLC (I always forget to change this too).

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Test the Prompt Creation Suite
+ */
+@RunWith(JUnit4.class)
```
Comment (jelares) [inline backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:31]: Maybe we should use JUnit5 for our testing? Will bring this up in the capstone daily today.

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    }
+    prompt+=keywords.get(keywords.size()-1);
```
Comment (aaronegolden) [inline backstory/src/main/java/com/google/sps/servlets/PromptManager.java:48]: Nit: consistent spacing "prompt += keywords..."

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        if(!outputPrompt.contains(keyword)){
+            actual = false;
+        }
+    }
+    
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void noKeywordInput() {
```
Comment (aaronegolden) [inline backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:67]: Might be good to also have a oneKeywordInput test to confirm we get (I think) whatever keyword followed by a period (Foo --> "Foo.").

Comment (UdayKalra) [reply backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:67]: Done

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java (hunk 5 above)
Comment (UdayKalra) [reply backstory/src/main/java/com/google/sps/servlets/PromptManager.java]: Done

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java (hunk 4 above)
Comment (UdayKalra) [reply backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:31]: Done

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java (hunk 3 above)
Comment (UdayKalra) [reply backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java]: Done

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java (hunk 2 above)
Comment (UdayKalra) [reply backstory/src/main/java/com/google/sps/servlets/PromptManager.java]: Done

File: backstory/src/main/java/com/google/sps/servlets/PromptManager.java (hunk 1 above)
Comment (UdayKalra) [reply backstory/src/main/java/com/google/sps/servlets/PromptManager.java:24]: Done

File: backstory/pom.xml [hunk 7]
```diff
@@ -65,6 +65,7 @@
       <version>2.6</version>
     </dependency>
 
+
```
Comment (jelares) [inline backstory/pom.xml:68]: NIT: Is there a need for this extra line?

File: backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java (hunk 4 above)
Comment (jelares) [reply backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java:31]: Sure, we can do that for simplicity of integration.

**Pull request merged**